velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -657,6 +664,22 @@ std::vector<std::string> AggregationFuzzer::generateKeys(
   return keys;
 }
 
+std::vector<std::string> AggregationFuzzer::generateSortingKeys(
+    const std::string& prefix,
+    std::vector<std::string>& names,
+    std::vector<TypePtr>& types) {
+  auto numKeys = boost::random::uniform_int_distribution<uint32_t>(1, 5)(rng_);
+  std::vector<std::string> keys;
+  for (auto i = 0; i < numKeys; ++i) {
+    keys.push_back(fmt::format("{}{}", prefix, i));
+
+    // Pick random, possibly complex, type.
+    types.push_back(vectorFuzzer_.randOrderableType(2));
```

Comment:
Orthogonal question : maxDepth 2 is good. Do you think there is value to make it configurable ? Guess more nesting doesn't really cover more code-paths..